docs/code-snippets/office-snippets.yaml

```diff
@@ -1817,6 +1817,13 @@ Office.Dialog#close:member(1):
             });
         }
     );
+Office.Dialog#messageChild:member(1):
+  - |-
+    // The following example shows how to send information about the current worksheet to the dialog.
+    function sheetPropertiesChanged() {
+        const messageToDialog = JSON.stringify({ name: "My Sheet", position: 2 });
```

Comment:
Something about using hard-coded worksheet data feels strange here. Feel free to disagree, but would it be more clear to have the `sheetPropertiesChanged` function take some parameters to pass to `stringify`.